onecodex/cli.py

```diff
@@ -169,7 +169,7 @@ def _size_formatter(size):
 @click.pass_context
 @pretty_errors
 @telemetry
-@login_required
+@login_required()
```

Comment:
This looks a bit awkward. Maybe the new parametrised wrapper could be called `login_required_args`
and then have `login_required = partial(login_required_args, experimental=False)` and `login_required_experimental_api = partial(login_required_args, experimental=True)`?